app/locale/ja.coffee

```diff
@@ -651,7 +651,7 @@ module.exports = nativeDescription: "日本語", englishDescription: "Japanese",
 #    already_have_account: "Already have a CodeCombat or Ozaria account"
 #    select_your_role: "Select your role"
 #    select_your_state: "Select your state"
-#    password_requirements: "4 to 64 characters with no repeating"
```

Comment:
i think u removed 1 extra space

Reply:
Oops! Sorry, fixed in https://github.com/codecombat/codecombat/pull/6633/commits/b373a9a9bb7ab936cc8c45524edb3ce8681a3866 🙇‍♂️